src/components/Loading/index.js

```diff
@@ -1,8 +1,8 @@
 import { Oval } from "react-loading-icons";
 
-export function Loading({ text = "Loading..." }) {
+export function Loading({ text = "Loading...", ...props }) {
   return (
-    <div className="flex justify-center items-center">
+    <div className={"flex justify-center items-center " + props.className}>
```

Comment:
Could use `clsx("flex justify-center items-center", props.className)`. otherwise if no className in props, it will show up as ` undefined` class.